lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -1341,6 +1341,15 @@ mapping:
           If this is null (the default), a simple configuration containing
           just Dockstore will be used.
 
+      beta_history_default_on:
```

Comment:
Can you set `GALAXY_CONFIG_BETA_HISTORY_DEFAULT_ON: true` in https://github.com/galaxyproject/galaxy/blob/dev/.github/workflows/selenium_beta.yaml#L10 ?

Reply:
Sure, will do.